packages/experiment-tag/src/experiment.ts

```diff
@@ -18,16 +19,27 @@ import {
   concatenateQueryParamsOf,
 } from './util';
 
+type WebExpUser = ExperimentUser & {
+  first_seen?: string;
+  device_type?: 'mobile' | 'tablet' | 'desktop';
```

Comment:
`device_type` is overloaded with the amplitude property. I would recommend changing this property name to be independent. maybe `device_category`?